style.css

```diff
@@ -177,26 +265,156 @@ a:link {
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
+  max-width: 35rem;
+  min-height: 36.9rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.form__list--small {
+  display: none;
+}
+
+.form__list--item > * {
+  width: 100%;
 }
 
 .works-project > *:not(:last-child) {
-  padding: 0 1.5rem;
+  padding: 0 1.6rem;
 }
 
 .works-project__languages {
-  list-style: none;
   display: flex;
   align-items: center;
   margin: 1rem 0 2.5rem 0;
 }
 
+.works-project__heading {
+  font-size: 3.2rem;
+  line-height: 4.4rem;
+  margin-top: 10rem;
+  max-width: 30rem;
+  font-weight: 400;
+  padding: 1rem 0;
+}
+
+.works-project__paragraph {
+  font-size: 2rem;
+  line-height: 2.8rem;
+  font-weight: 400;
+  max-width: 35rem;
+}
+
 .works-stories__btn--link {
   font-size: 1.7rem;
   font-weight: 600;
-  padding: 1rem 2rem;
+  padding: 1.8rem 2.5rem;
   background-color: #ff6b00;
 }
 
+.works-project__languages--item {
+  padding: 0.5rem 1.2rem;
+  font-size: 1.5rem;
+  background-color: rgba(255, 255, 255, 0.24);
+  margin-right: 1rem;
+}
+
+.works-project__languages--item:hover {
+  cursor: pointer;
+}
+
+@media only screen and (min-width: 476px) {
+  .works-stories {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  .works-stories .right {
+    width: 34%;
+    margin-left: 2rem;
+  }
+
+  .works-stories__photo {
+    width: 60%;
+  }
+
+  .works-project__btn {
+    display: none;
+  }
+
+  /*** works project background ***/
+  .works-project-1 {
+    background-image: url('./img/project-1.jpg');
+  }
+
+  .works-project-1 .works-project__btn {
+    display: block;
+  }
+
+  .works-project-1 > * {
+    display: none;
+  }
+
+  .works-project-2 {
+    background-image: url('./img/project-2.jpg');
+  }
+
+  .works-project-3 {
+    background-image: url('./img/project-3.jpg');
+  }
+
+  .works-project-4 {
+    background-image: url('./img/project-4.jpg');
+  }
+
+  .works-project-5 {
+    background-image: url('./img/project-5.jpg');
+  }
+
+  .works-project-6 {
+    background-image: url('./img/project-6.jpg');
+  }
```

Comment:
Great job implementing the UI design but there is a single issue, I know it was not so clear. 
- [ ] This portfolio template project is supposed to look like your last 5 projects and ONLY when hovered the description, title, and technologies are disappeared and the button is displayed (like your first example)

Reply:
I thought we weren't suppose to implement animation and transitions yet.